examples/synthea-breast-cancer/Ali_Laiba_syntheaEDA/SyntheaClassifiers.ipynb

```diff
@@ -0,0 +1,1710 @@
+       "1398            16  \n",
+       "1399            10  \n",
+       "\n",
+       "[1400 rows x 8 columns]"
+      ]
+     },
+     "execution_count": 6,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "df"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "**Creating Input and Output Datasets**"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "X = []\n",
+    "y = []\n",
+    "\n",
+    "for index, row in df.iterrows():\n",
+    "    X.append([row.surgical, row.radiation, row.cancerStatus, row.diagnosisAge, row.primary, row.nodes, row.numberOfMeds])\n",
+    "    y.append(row.stage)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "**Splitting the Data into training and testing Datasets**\n",
+    "\n",
+    "There is no need for the random_state parameter to be 1729 specifically. We followed Rishabh's [Non-Federated Logistic Regression on the Synthea MCODE Breast Cancer Dataset](https://github.com/CanDIG/federated-learning/blob/main/examples/synthea-breast-cancer/Non-FederatedClassification.ipynb) Jupyter Notebook when training our datasets for the first time and said notebook used 1729 as its random state. The random_state parameter is useful in making sure there is consistency between team member's when they are analysing data, but the specific value may be changed. We've also made two copies of each of the datasets, one to be used with scaling and one without."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "X_train, X_test, y_train, y_test = train_test_split(X, y, test_size=0.2, random_state=1729)\n",
+    "\n",
+    "X_train_no_scale, X_test_no_scale = X_train.copy(), X_test.copy()\n",
+    "y_train_no_scale, y_test_no_scale = y_train.copy(), y_test.copy()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "**Viewing Sample Training and Testing Data**"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 9,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "TRAINING DATA SAMPLE\n",
+      "INPUTS: [2.0, 34.0, 1.0, 38.97330595482546, 2.0, 1.0, 2.0]\n",
+      "OUTPUT: 2.0\n",
+      "\n",
+      "TESTING DATA SAMPLE\n",
+      "INPUTS: [1.0, 34.0, 1.0, 54.96235455167693, 1.0, 0.0, 3.0]\n",
+      "OUTPUT: 1.0\n"
+     ]
+    }
+   ],
+   "source": [
+    "print('TRAINING DATA SAMPLE')\n",
+    "print(f'INPUTS: {X_train[0]}')\n",
+    "print(f'OUTPUT: {y_train[0]}')\n",
+    "print()\n",
+    "\n",
+    "print('TESTING DATA SAMPLE')\n",
+    "print(f'INPUTS: {X_test[0]}')\n",
+    "print(f'OUTPUT: {y_test[0]}')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "**Scaling and Transforming Data**"
```

Comment:
Not sure what is being accomplished in this block, could you add some narrative text to describe what this code does?